api/build.gradle

```diff
@@ -698,6 +698,16 @@ task exportWorkspaceData(type: JavaExec) {
   }
 }
 
+// See project.rb command: delete-workspaces
```

Comment:
Can I also call this with gradle directly?

Reply:
Wouldn't recommend it. You need the cloud sql database proxy that project.rb will set up for you.

Reply:
interesting.